### PR TITLE
Rename poc_ingest to ingest

### PR DESCRIPTION
**Pull request by bbalser**

### CODE REVIEW

**Review by jeffgrunewald — COMMENTED** [review]
2 questions since this will likely come up eventually:
- do we want to rename the operating modes for the ingest server from `5g` and `lora` to `mobile` and `iot`?
- do these python test scripts still work such that they're worth renaming or are they so outdated they should just be deleted instead?

**Comment by madninja** [comment]
> 2 questions since this will likely come up eventually:
> 
> * do we want to rename the operating modes for the ingest server from `5g` and `lora` to `mobile` and `iot`?

that’d be great. Yes please

File: ingest/src/settings.rs [hunk 1]
```diff
@@ -49,7 +49,7 @@ pub fn default_sink() -> String {
 #[derive(Debug, Deserialize)]
 #[serde(rename_all = "lowercase")]
 pub enum Mode {
-    Lora,
+    IOT,
```
Comment (madninja) [inline ingest/src/settings.rs:52]: I'm not sure the all caps `IOT` works. I think we should just use `Iot` to avoid  enum case clippy issues

Comment (jeffgrunewald) [reply ingest/src/settings.rs:52]: + 1

Comment (bbalser) [reply ingest/src/settings.rs]: Fixed d9660c6

**Review by jeffgrunewald — APPROVED** [review]

**Review by madninja — APPROVED** [review]

**Pull request merged**